### ISSUE TITLE
באג 1000 - ⚠️❗

### ISSUE THREAD

**Issue by NetanelHarris** [kdroidFilter/HebrewNumeralsLibrary#1]
אני לא בטוח שזה נכון לקרוא לזה שגיאה כי אולי זה ההתנהגות הנכונה, אבל שים לב שגם 1 וגם 1000 זה א', ובגימטריה זה 1, אז איך כותבים 1000 בגימטריה?

**Comment by kdroidFilter** [comment]
נכון, לא חשבתי על זה, צריך לחשוב על צורת כתיבה אחרת בשביל מספרים עגולים
